crates/oct-py/python/opencloudtool/py_api.py

```
@@ -64,3 +64,78 @@ def destroy(path: str = "."):
+def deploy_service(path: str) -> None:
+    """
+    Automates the deployment of a standard FastAPI service.
+
+    This high-level function generates the necessary Dockerfile, requirements.txt,
+    and oct.toml, then calls the core deploy function.
+
+    Args:
+        path: The path to the FastAPI application's main.py file.
+    """
+    if not os.path.isfile(path) or not path.endswith("main.py"):
+        raise ValueError("The path must point to a 'main.py' file.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)